react-common/components/controls/Feedback/Feedback.tsx

```diff
@@ -59,12 +66,10 @@ export const FeedbackModal = (props: IFeedbackModalProps) => {
     }
   }, [])
   return (
-    <Modal className="feedback-modal" title={title} onClose={onClose}>
       <iframe
       title="feedback"
       id={frameId}
-      src={`${pxt.appTarget.appTheme.ocvFrameUrl}/centrohost?appname=ocvfeedback&feature=host-ocv-inapp-feedback&platform=web&appId=${pxt.appTarget.appTheme.ocvAppId}#/hostedpage`}
+      src={`${pxt.webConfig.ocv?.iframeEndpoint}/centrohost?appname=ocvfeedback&feature=host-ocv-inapp-feedback&platform=web&appId=${pxt.webConfig.ocv?.appId}#/hostedpage`}
```

Comment:
I'm wondering if, rather than `ocv?.`, we should we check `src={ocv ? "..." : undefined}`. What actually happens if ocv is null here and this becomes a malformed URL?

Reply:
Oh, that's an interesting idea. Technically, in order to have this component be active at all it has to go through the menus where we already have checks to make sure that the ocv object exists. We might just want to do a check for the ocv object before the iframe in this case, because I agree, we wouldn't want a malformed url here.